### PR TITLE
kommander: Add workspace platform service resource requirements page

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-73733

## Description of changes being made
Add new page with workspace platform services resource requirements table. The table can be more easily viewed in this google doc https://docs.google.com/document/d/148tdEFt6iycMDRe5kcbMDxQtkHqGjicH2AN1hLSKQdA/edit?usp=sharing feel free to also make comments in that doc

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.3/workspaces/workspace-platform-services/index.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The following tables describe the list of platform services and cluster platform
 
 Currently, the monitoring stack is federated by default and the logging stack is not.
 
+You should also review the [workspace platform service resource requirements](/dkp/kommander/1.3/workspaces/platform-service-requirements/) to ensure that the attached clusters have sufficient resources.
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/workspaces/workspace-platform-services/index.md:24]:
```suggestion
Review the [workspace platform service resource requirements](/dkp/kommander/1.3/workspaces/platform-service-requirements/) to ensure that the attached clusters have sufficient resources.
```
making the same change here @cmastr

**Comment by cmastr** [comment]
@gracedo this looks good. Why the .tmpl though? Will we reuse this?

**Comment by gracedo** [comment]
> @gracedo this looks good. Why the .tmpl though? Will we reuse this?

@cmastr good question, I was just following the pattern from kba-requirements.tmpl from the konvoy docs, but I don't think we need to reuse it so feel free to move that out of include and directly into the page!

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
> > @gracedo this looks good. Why the .tmpl though? Will we reuse this?
> 
> @cmastr good question, I was just following the pattern from kba-requirements.tmpl from the konvoy docs, but I don't think we need to reuse it so feel free to move that out of include and directly into the page!

oh! this is a reuse of a template somewhere else? If we think this will get reused, then we'll keep it a template.

**Comment by gracedo** [comment]
> > > @gracedo this looks good. Why the .tmpl though? Will we reuse this?
> > 
> > 
> > @cmastr good question, I was just following the pattern from kba-requirements.tmpl from the konvoy docs, but I don't think we need to reuse it so feel free to move that out of include and directly into the page!
> 
> oh! this is a reuse of a template somewhere else? If we think this will get reused, then we'll keep it a template.

@cmastr ah, no this isn't reused, there is just a similar template elsewhere but in both that and this cases, the template is only used in one place, so maybe we should take out the templates

**Pull request merged**